test/common/build/controllers.go

```diff
@@ -653,6 +649,10 @@ func RunBuildRunningPodDeleteTest(t testingT, clusterAdminClient *client.Client,
 	if newBuild.Status.Phase != buildapi.BuildPhaseError {
 		t.Fatalf("expected build status to be marked error, but was marked %s", newBuild.Status.Phase)
 	}
+
+	// allow time for the failed event to get created
+	time.Sleep(5 * time.Second)
```

Comment:
> Is the above one straight removable now the poddeletecontroller has gone?

We still handle the pod delete event and update the build accordingly. The comment should be updated though.